src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -100,6 +100,19 @@
                     :formatFunction="formatBarchartValue"
                     :showRangesInTooltips="true"
                     @update="updateComparisonPlotSelectionsAndXAxisOrder"></bar-chart-with-filters>
+                <div class="row mt-2">
+                    <div class="col-md-3"></div>
+                    <area-indicators-table class="col-md-9"
+                                        :table-data="comparisonPlotData"
+                                        :area-filter-id="areaFilterId"
+                                        :filters="comparisonPlotFilters"
+                                        :countryAreaFilterOption="countryAreaFilterOption"
+                                        :indicators="filteredComparisonPlotIndicators"
+                                        :selections="comparisonPlotSelections"
+                                        :translate-filter-labels="false"
+                                        :selectedFilterOptions="comparisonPlotSelections.selectedFilterOptions"
+                    ></area-indicators-table>
+                </div>
                 <error-alert v-if="!!comparisonPlotError" :error="comparisonPlotError"></error-alert>
```

Comment:
Alternatively, the error alert could go above the table?

Reply:
Presumably if there was an error there wouldn't be any data in the table, in which case it wouldn't really matter since the table wouldn't show anything. Or could it be that we have data hanging around from previous fetch which doesn't get cleared when error on next fetch is committed? Should either clear old data or put error somewhere definitely visible.